mahta/JS/JS data fetching/script.js

```diff
@@ -0,0 +1,196 @@
+function createPostElement() {
```

Comment:
It is required to optimize the function to create an element tag efficiently without creating all post and body code.